pkg/sdn/api/v1/types.go

```diff
@@ -35,3 +35,19 @@ type HostSubnetList struct {
 	kapi.ListMeta `json:"metadata,omitempty"`
 	Items         []HostSubnet `json:"items" description:"list of host subnets"`
 }
+
+// NetNamespace encapsulates the inputs needed to define a unique network namespace on the cluster
+type NetNamespace struct {
+	kapi.TypeMeta   `json:",inline"`
+	kapi.ObjectMeta `json:"metadata,omitempty"`
+
+	NetName string `json:"host" description:"Name of the network namespace."`
+	NetID   uint   `json:"netid" description:"NetID of the network namespace assigned to each overlay network packet."`
```

Comment:
The only type supported by json is int64, so you'll have to check for a positive integer is correct during validation